balance/controllers/address.js

```diff
@@ -14,6 +14,8 @@ BigNumber.config({ EXPONENTIAL_AT: 28 })
 const debug = new SDebug('balance')
 const v2 = {}
 
+let parameters = {}
```

Comment:
this won't work if we're running multiple workers - which we are in prod

Reply:
(we'll only update one worker's copy)

Reply:
should be fixed!